modules/installation-aws-about-government-region.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="installation-aws-about-government-region_{context}"]
+= AWS government regions
+
+{product-title} supports deploying a cluster to
+link:https://aws.amazon.com/govcloud-us[AWS GovCloud (US)] regions. AWS GovCloud
+is specifically designed for US government agencies at the federal, state, and
+local level, as well as contractors, educational institutions, and other US
+customers that must run sensitive workloads in the cloud.
+
+These regions do not have published {op-system-first} AMIs to select, so you
```

Comment:
s/AMIs/Amazon Machine Images (AMI)

If you choose to add this one, there are a couple of other modules that mention AMIs :)